ros/src/twist_controller/twist_controller.py

```diff
@@ -42,6 +42,13 @@ def control(self, current_vel, dbw_enabled, linear_vel, angular_vel):
 
         steering = self.yaw_controller.get_steering(linear_vel, angular_vel, current_vel)
 
+        # Write to ros log
```

Comment:
why it's commented?

Reply:
sry It's a mistake. I will re-pull another version.

Reply:
you can just commit and push to the same branch :)